### ISSUE TITLE
Hidden fields do not have titles in the query field

### ISSUE THREAD

**Issue by Minoru** [newsboat/newsboat#636]
_Originally posted by @NilsIrl in https://github.com/newsboat/newsboat/issues/33#issuecomment-528454050_

Another problem is that hidden fields do not have titles in the query field.

Let me illustrate:

# Without hidden flag

```
https://www.youtube.com/feeds/videos.xml?channel_id=UCS0N5baNlQWJCUrhCEo8WlA youtube
https://www.youtube.com/feeds/videos.xml?channel_id=UCRcgy6GzDeccI7dkbbBna3Q youtube
https://www.youtube.com/feeds/videos.xml?channel_id=UCsXVk37bltHxD1rDPwtNM8Q youtube
https://www.youtube.com/feeds/videos.xml?channel_id=UCH6ppHEvV3_WIXEwmhv9HEg youtube
https://www.youtube.com/feeds/videos.xml?channel_id=UCCezIgC97PvUuR4_gbFUs5g youtube
https://www.youtube.com/feeds/videos.xml?channel_id=UCYpRDnhk5H8h16jpS84uqsA youtube
https://www.youtube.com/feeds/videos.xml?channel_id=UCXuqSBlHAE6Xw-yeJA0Tunw youtube
https://www.youtube.com/feeds/videos.xml?channel_id=UCXPHFM88IlFn68OmLwtPmZA youtube
https://www.youtube.com/feeds/videos.xml?channel_id=UCi8XrDg1bK_MJ0goOnbpTMQ youtube
https://www.youtube.com/feeds/videos.xml?channel_id=UCCkVMojdBWS-JtH7TliWkVg youtube
https://www.youtube.com/feeds/videos.xml?channel_id=UCeeFfhMcJa1kjtfZAGskOCA youtube
https://www.youtube.com/feeds/videos.xml?channel_id=UCOWcZ6Wicl-1N34H0zZe38w youtube
https://www.youtube.com/feeds/videos.xml?channel_id=UC2eYFnH61tmytImy1mTYvhA youtube
https://www.youtube.com/feeds/videos.xml?channel_id=UClcE-kVhqyiHCcjYwcpfj9w youtube
https://www.youtube.com/feeds/videos.xml?channel_id=UC9-y-6csu5WGm29I7JiwpnA youtube
https://www.youtube.com/feeds/videos.xml?channel_id=UCwRXb5dUK4cvsHbx-rGzSgw youtube
https://www.youtube.com/feeds/videos.xml?channel_id=UCzuaB4F2znrMggxcwUuVhAw youtube

"query:Youtube:tags # \"youtube\""
```

![screenshot](https://user-images.githubusercontent.com/26231126/64361070-b44dfd00-d003-11e9-8632-13be1958972a.png)

# With hidden flag

```
https://www.youtube.com/feeds/videos.xml?channel_id=UCS0N5baNlQWJCUrhCEo8WlA ! youtube
https://www.youtube.com/feeds/videos.xml?channel_id=UCRcgy6GzDeccI7dkbbBna3Q ! youtube
https://www.youtube.com/feeds/videos.xml?channel_id=UCsXVk37bltHxD1rDPwtNM8Q ! youtube
https://www.youtube.com/feeds/videos.xml?channel_id=UCH6ppHEvV3_WIXEwmhv9HEg ! youtube
https://www.youtube.com/feeds/videos.xml?channel_id=UCCezIgC97PvUuR4_gbFUs5g ! youtube
https://www.youtube.com/feeds/videos.xml?channel_id=UCYpRDnhk5H8h16jpS84uqsA ! youtube
https://www.youtube.com/feeds/videos.xml?channel_id=UCXuqSBlHAE6Xw-yeJA0Tunw ! youtube
https://www.youtube.com/feeds/videos.xml?channel_id=UCXPHFM88IlFn68OmLwtPmZA ! youtube
https://www.youtube.com/feeds/videos.xml?channel_id=UCi8XrDg1bK_MJ0goOnbpTMQ ! youtube
https://www.youtube.com/feeds/videos.xml?channel_id=UCCkVMojdBWS-JtH7TliWkVg ! youtube
https://www.youtube.com/feeds/videos.xml?channel_id=UCeeFfhMcJa1kjtfZAGskOCA ! youtube
https://www.youtube.com/feeds/videos.xml?channel_id=UCOWcZ6Wicl-1N34H0zZe38w ! youtube
https://www.youtube.com/feeds/videos.xml?channel_id=UC2eYFnH61tmytImy1mTYvhA ! youtube
https://www.youtube.com/feeds/videos.xml?channel_id=UClcE-kVhqyiHCcjYwcpfj9w ! youtube
https://www.youtube.com/feeds/videos.xml?channel_id=UC9-y-6csu5WGm29I7JiwpnA ! youtube
https://www.youtube.com/feeds/videos.xml?channel_id=UCwRXb5dUK4cvsHbx-rGzSgw ! youtube
https://www.youtube.com/feeds/videos.xml?channel_id=UCzuaB4F2znrMggxcwUuVhAw ! youtube

"query:Youtube:tags # \"youtube\""
```

![screenshot](https://user-images.githubusercontent.com/26231126/64361146-ecedd680-d003-11e9-9db2-4261cccb8db8.png)

This is very annoying.

**Comment by NilsIrl** [comment]
Sorry for the annoyance. But I was running 1.14 (from my package manager).

this was fixed in 4e0d4a37b134a6b3183429624ade78e4358e1c85.

**Comment by Minoru** [comment]
My favourite kind of bugs, the fixed ones :) Thanks for notifying.
